sre-recipes/sandboxctl

```diff
@@ -41,11 +41,13 @@ def get_valid_recipes(attributes):
     """
 
     recipe_objs = {}
-    for recipe_num, (attribute_name, attribute) in enumerate(attributes):
+    recipe_num = 0
+    for (attribute_name, attribute) in attributes:
```

Comment:
This works. Another (potentially cleaner) way would be to filter the list before iteration rather than as part of it:

```
filtered = [a for a in attributes if isclass(a) and a is not Recipe and issubclass(a, Recipe)]
for recipe_num, (attribute_name, attribute) in enumerate(filtered):
```
But I don't think it matters too much either way

Reply:
And I guess if the TypeErrors are expected, your way might be safer since it only counts recipes that it knows work

Reply:
ah yeah that's true - I think we can keep it as is then!